raiden/tests/integration/api/rest/test_rest.py

```diff
@@ -788,7 +822,7 @@ def test_payment_events_endpoints(
         json_response,
         {
             "event": "EventPaymentSentSuccess",
-            "identifier": str(identifier3),
+            "identifier": str(identifier4),
```

Comment:
This is missing the `EventPaymentReceivedSuccess` but with `identifier3`

Reply:
good catch